docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-my-account-feature-integration.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Merchant Portal My Account feature integration
```

Comment:
We have IGs for Spryker features, not for epic. the corresponding IGs should be updated